website/_config.yml

```diff
@@ -53,6 +53,10 @@ documentations:
         url: /docs/tutorial/classification.html
       - title: Eagle LDAP Tutorial
         url: /docs/tutorial/ldap.html
+  - category: Developer
```

Comment:
Good doc, I would prefer to Change to category name to `Development` instead of `Developer`, how do you think?